src/store/getters/state-getters.ts

```diff
@@ -286,12 +296,26 @@ export const isFilingPaying = (state: StateIF): boolean => {
   return state.stateModel.tombstone.isFilingPaying
 }
 
-/** Whether any correction/alteration sections have changed. */
-export const isFilingChanged = (state: StateIF): boolean => {
-  return (state.stateModel.peopleAndRolesStep.changed ||
+/** Whether any correction data has changed. */
+export const hasCorrectionChanged = (state: StateIF): boolean => {
+  return (
+    state.stateModel.peopleAndRolesStep.changed ||
     state.stateModel.defineCompanyStep.changed ||
     state.stateModel.shareStructureStep.changed ||
-    state.stateModel.incorporationAgreementStep.changed)
+    state.stateModel.incorporationAgreementStep.changed
+  )
+}
+
+/** Whether any alteration fee-required data has changed. */
+export const hasAlterationChanged = (state: StateIF): boolean => {
+  return (
+    hasBusinessNameChanged(state) ||
+    hasBusinessTypeChanged(state) ||
+    hasNameTranslationChanged(state) ||
+    hasShareStructureChanged(state) ||
+    getProvisionsRemoved(state) ||
+    hasNewResolutionDatesChanged(state)
```

Comment:
If you worked on an alteration component, please verify that it's correctly handled here.

I did this based on what I saw in the UI but there may be bits missing.

Reply:
There is one optional field in the document delivery section... do you think we need to add the certify section here as well?

Reply:
Can the document delivery section be incomplete or invalid in any way?

Reply:
invalid email address

Reply:
Good catch!

@cameron-freshworks Can you add the validation check for this? I will add a getter that you can use to check if valid.

Reply:
The new getter will check if DocumentsDelivery has changed. Sorry, nothing to do with validation.